spec_macros/src/fix_enum_serialization.rs

```diff
@@ -0,0 +1,614 @@
+// Copyright (c) 2025 Elektrobit Automotive GmbH
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache License, Version 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+use proc_macro2::TokenStream;
+use quote::{
+    //ToTokens,
+    format_ident,
+    quote,
+};
+use syn::{Attribute, Data, DeriveInput, Fields, FieldsUnnamed, Type};
+
+use crate::utils;
```

Comment:
Same logic with the imports.
```suggestion
// SPDX-License-Identifier: Apache-2.0

use crate::utils;

use proc_macro2::TokenStream;
use quote::{
    format_ident,
    quote,
};
use syn::{Attribute, Data, DeriveInput, Fields, FieldsUnnamed, Type};
```